website/search/views.py

```diff
@@ -51,6 +48,16 @@ def wrapped(*args, **kwargs):
                 'message_long': ('Our search service is currently unavailable, if the issue persists, '
                 'please report it to <a href="mailto:support@osf.io">support@osf.io</a>.'),
             })
+        except exceptions.SearchException:
+            # Interim fix for issue where ES fails with 500 in some settings- ensure exception is still logged until it can be better debugged. See OSF-4538
+            sentry.log_exception()
+            sentry.log_message('Elasticsearch returned an unexpected error response')
+            # TODO: Add a test; may need to mock out the error response due to inability to reproduce error code locally
+            raise HTTPError(http.BAD_REQUEST, data={
+                'message_short': 'Could not perform search query',
+                'message_long': ('Please check our help (the question mark beside the search box) for more information '
+                                 'on advanced search queries.'),
```

Comment:
Is this really the long message we want to give folks in this case? If it is, then we could/should [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up with the `MalformedQueryError` case (on the pattern of our [underlying](https://github.com/CenterForOpenScience/osf.io/blob/c71d014da1739fd6858bfde5d7c3bac5f730241b/website/search/elastic_search.py#L84-L87) `RequestError` ([&rarr; `TransportError`](https://github.com/CenterForOpenScience/osf.io/pull/5364#discussion_r58397540)) handling, since `MalformedQueryError` subclasses `SearchException`). If not, then we should change it. :-)